TAF_EasyRest/TestFramework/PageComponents/AdministratorPanelComponents/AcceptedCardPageComponent.cs

```diff
@@ -0,0 +1,35 @@
+﻿
+namespace TestFramework.PageComponents.AdministratorPanelComponents
+{
+    public class AcceptedCardPageComponent
+    {
+        IWebDriver driver { get; }
+
+        public AcceptedCardPageComponent(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+
+        private IWebElement _selectWaiterRadioButton => driver.FindElement(By.XPath("(//input)[1]"));
+        private IWebElement _assignButton => driver.FindElement(By.XPath("(//span[text()='Assign'])[1]"));
+
+        public AcceptedCardPageComponent SelectAnyWaiter()
```

Comment:
please rename to Select**TheFirst**Waiter()

Reply:
done